scripts/helper.js

```diff
@@ -1061,7 +1059,7 @@ HELPER.test = (() => {
 						error += `The module ${ module } is missing "react" as devDependency.\n`;
 					}
 
-					if( hasReact && Object.keys( packagesPKG.devDependencies ).length < 12 ) {
+					if( hasReact && Object.keys( packagesPKG.devDependencies ).length < 10 ) {
```

Comment:
we changed this because babel is now one package right?

Reply:
Yeah we no longer need `transform-spread...` and the other one.